lib/management/configuration.js

```diff
@@ -183,6 +183,19 @@ function patchConfiguration(newConf, log, cb) {
                         };
                     }
                     break;
+                case 'location-nfs-mount-v1':
+                    location.type = 'pfs';
+                    if (l.details) {
+                        location.details = {
+                            supportsVersioning: false,
+                            bucketMatch: true,
+                            pfsDaemonEndpoint: {
+                                host: l,
```

Comment:
shouldn't it be something like `l.name` + `cosmos-psd`?

Reply:
Yeah, my bad. Although that won't work if the location name is `cosmos`.

Reply:
Yeah it I was just thinking about that too because of the templating on the chart. What would the env var define?

Reply:
Yeah, just the suffix. We can't still have a location called cosmos though.